### ISSUE TITLE
[Feature request] Proper health check  BOUNTY 210k sat

### ISSUE THREAD

**Issue by Opago-Pay** [lnbits/lnbits#2735]
**Is your feature request related to a problem? Please describe.**
The health check endpoint currently returns 200 basically always, even if the node connection is lost. This makes it useless.

**Describe the solution you'd like**
Ideally the health check endpoint would return proper status information to be ingested e.g. bei Prometheus and other monitoring software.

**Describe alternatives you've considered**
For the meantime, we’d be happy if the endpoint simply checked the node connection and whether invoice creation is possible. For this we will pay 210k sat to whoever implements the PR, once that PR is merged into a release.

**Comment by motorina0** [comment]
PR here: https://github.com/lnbits/lnbits/pull/2736

**Comment by prusnak** [comment]
Fixed via https://github.com/lnbits/lnbits/pull/2736
